### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by omerbasoglu-co**

### CODE REVIEW

**Comment by omerbasoglu-co** [comment]
I built for the #41 @yakuter could you please code review

File: pkg/router/router.go [hunk 1]
```diff
@@ -37,15 +37,29 @@ func Setup() *gin.Engine {
 		auth.POST("/refresh", authMW.MiddlewareFunc(), authMW.RefreshHandler)
 	}
 
+	asa := r.Group("/asa", authMW.MiddlewareFunc())
```
Comment (yakuter) [inline pkg/router/router.go]: I didn't understand this asa router group. Do we need this?

File: login/login_api.go [hunk 2]
```diff
@@ -20,6 +20,22 @@ func NewLoginAPI(p LoginService) LoginAPI {
 	return LoginAPI{LoginService: p}
 }
 
+// FindSamePassword ...
+func (p *LoginAPI) FindSamePassword(c *gin.Context) {
+
+	password := c.PostForm("password")
```
Comment (yakuter) [inline login/login_api.go:26]: We use JSON in HTTP request body between clients and server commonunication. Can you change this to:
```
...
var password Password
c.BindJSON(&password)
...
```

File: pkg/router/router.go (hunk 1 above)
Comment (omerbasoglu-co) [reply pkg/router/router.go]: I forgotten remove the block when I test, I use. I fixed [e8d02d9](https://github.com/pass-wall/passwall-server/pull/54/commits/e8d02d94beabd45576a576440e36df8fda6ec6cd)

File: login/login_api.go (hunk 2 above)
Comment (yakuter) [inline login/login_api.go:26]: We use JSON in HTTP request body between clients and server commonunication. Can you change this to:
```

...
var password Password
c.BindJSON(&password)
...
```

File: login/url_dto.go [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+type Url struct {
+	Name string `json:"url"`
+}
+
+type Urls struct {
+	Items []Url `json:"urls"`
+}
+
+func (urls *Urls) AddItem(item Url) {
```
Comment (yakuter) [inline login/url_dto.go:11]: Can we change these lines to make return object like this:
```
    "URLs": [
             "hepsiburada.com",
             "abc.com",
             "cfg.com",
    ]
```

Comment (omerbasoglu-co) [reply login/url_dto.go:11]: ı think we may extend struct such as add favorite field, if you want again after you think , ı can do 

url | favorite
d1 | *
d3 | *
d2
d4
d6

File: login/login_api.go (hunk 2 above)
Comment (omerbasoglu-co) [reply login/login_api.go]: I just completed [c685435](https://github.com/pass-wall/passwall-server/pull/54/commits/c685435596dfa79e34c515a2ce9878bf517f96c4)

File: login/url_dto.go (hunk 3 above)
Comment (yakuter) [reply login/url_dto.go:11]: Thank you but I am really trying hard to keep it simple. Maybe in the future we should add extra features to compete with other companies but for now I think we don't need it.

**Comment by yakuter** [comment]
Thank @omerbasoglu-co for your contribution. This was really helpful.

File: login/url_dto.go (hunk 3 above)
Comment (omerbasoglu-co) [reply login/url_dto.go:11]: I can understand you :) I m doing :)

**Comment by yakuter** [comment]
Hello @omerbasoglu-co I just wanted to show the result of your work :D

80291125-83b60180-8753-11ea-93c8-2b9f74368986

**Comment by omerbasoglu-co** [comment]
thanks @yakuter it looks very good! 
<lang:tr>Eline sağlık abi!</lang:tr>

**Pull request merged**